.github/workflows/ci.yml

```diff
@@ -12,13 +12,12 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'min'
+          - 'lts'
           - '1' # automatically expands to the latest stable 1.x release of Julia
-          - 'nightly'
+          - 'pre'
         os:
           - ubuntu-latest
-          - macOS-latest
-          - windows-latest
```

Comment:
I think we need the tests on 32-bit machines. Does `ubuntu-latest` include them?

Reply:
OK - I see it does, so I think it is OK.